spec/lib/category_guardian_spec.rb

```diff
@@ -64,4 +64,59 @@
       end
     end
   end
+
+  describe "topics_need_approval?" do
+    fab!(:reviewable_group) { Fabricate(:group) }
+
+    it "returns false when admin" do
+      expect(Guardian.new(admin).topics_need_approval?(category)).to eq(false)
+    end
+
+    it "returns the value of require_topic_approval when group moderation is off" do
```

Comment:
Minor but I don't think we should mention `require_topic_approval` in the specs because this is internal implementation detail.